docs/modular-resources/upload/_index.md

```diff
@@ -30,21 +30,37 @@ To upload your custom module to the [Viam registry](https://app.viam.com/registr
 
 1. First, [install the Viam CLI](/manage/cli/#install) and [authenticate](/manage/cli/#authenticate) to Viam, from the same machine that you intend to upload your module from.
 
-1. Next, run the `viam module create` command to select a new custom module name and generate module metadata.
+2. Next, run the `viam module create` command to select a new custom module name and generate module metadata.
+   Run this command according to your desired visibility for your module:
```

Comment:
I think this would be a good place to add that visibility is private by default (i.e. just before the choice of which to select)

Reply:
@andf-viam Edited, sounds a bit awkward with the flow but I can't think of a better point to introduce